test/webrick/test_httprequest.rb

```diff
@@ -632,6 +634,25 @@ def test_eof_raised_when_line_is_nil
     }
   end
 
+  def test_eof_raised_with_missing_line_between_headers_and_body
+    msg = <<-_end_of_message_
+      GET / HTTP/1.0
+    _end_of_message_
+    req = WEBrick::HTTPRequest.new(WEBrick::Config::HTTP)
+    assert_raise(WEBrick::HTTPStatus::EOFError) {
+      req.parse(StringIO.new(msg.gsub(/^ {6}/, "").gsub("\n", "\r\n")))
```

Comment:
Do you mind preferring squigly heredocs?

Reply:
Not at all.  I pushed a rebased commit that adds squigly heredocs.  It also has a test change to a recently merged test.  I think the test change is reasonable, but let me know your thoughts.